i18n/en.yaml

```diff
@@ -506,11 +506,14 @@ FALLOUT.UI.CONSUMABLES: Consumables
 FALLOUT.UI.Consume: Consume
 FALLOUT.UI.Cost: Cost
 FALLOUT.UI.Damage: Damage
+FALLOUT.UI.NUMOFDTOREROLLLUCK: How many dice you want to re-roll using luck
+FALLOUT.UI.NUMOFDTOREROLL: How many dice you want to re-roll not using luck
```

Comment:
These keys are not very readable.  Going forward I'm trying to standardise the i18n key style to snake case, avoid unecessary abbreviations where possible, so could you change these to something like:

`FALLOUT.UI.reroll_with_luck_count` and `fALLOUT.UI.reroll_count`

Reply:
done